test-framework/security-jwt/pom.xml

```diff
@@ -28,6 +28,10 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jsonp</artifactId>
```

Comment:
Not a big deal because it's for testing, but I wonder if this dependency can be made optional